Modules/posixmodule.c

```
@@ -17556,11 +17556,11 @@ posixmodule_exec(PyObject *m)
             return -1;
         }
 
-        if (PyDict_DelItemString(dct, "pwritev") == -1) {
-            PyErr_Clear();
+        if (PyDict_PopString(dct, "pwritev", NULL) < 0) {
+            return -1;
         }
-        if (PyDict_DelItemString(dct, "preadv") == -1) {
-            PyErr_Clear();
+        if (PyDict_PopString(dct, "preadv", NULL) < 0) {
+            return -1;
```

This is slightly different behavior, isn't it?  Before, we tried removing each item and ignored *all* errors.  Now we fail at the first error.  Is that okay?  (Maybe @ronaldoussoren has some thoughts on this?)  Was this intentional?

(The same applies to the changes in Modules/timemodule.c.)

---

Not ignoring errors is always a good thing.